package.json

```diff
@@ -5,7 +5,8 @@
   "description":  "Client for interfacing taskcluster components",
   "license":      "MPL-2.0",
   "scripts": {
-    "test":       "./test/runtests.sh"
+    "test":       "./test/runtests.sh",
+    "compile":      "babel lib --out-dir build"
```

Comment:
@imbstack, what's the trick to setup `babel-compile` with `babel-taskcluster-preset` ?